community_supplied/Kernel/Multicast/multicast-iif-mismatch.playbook

```diff
@@ -0,0 +1,5 @@
+iceberg {
```

Comment:
As we discussed
- add the rules functionality in all playbooks
- replace iceberg with healthbot in all rule and playbook filed